iceberg/iceberg-handler/src/test/results/positive/query_iceberg_metadata_of_partitioned_table.q.out

```diff
@@ -341,6 +341,134 @@ POSTHOOK: type: QUERY
 POSTHOOK: Input: default@ice_meta_3
 POSTHOOK: Output: hdfs://### HDFS PATH ###
 [{"contains_null":false,"contains_nan":false,"lower_bound":"four","upper_bound":"two"},{"contains_null":false,"contains_nan":false,"lower_bound":"Friday","upper_bound":"Wednesday"}]
+PREHOOK: query: select * from default.ice_meta_2.data_files
+PREHOOK: type: QUERY
+PREHOOK: Input: default@ice_meta_2
+PREHOOK: Output: hdfs://### HDFS PATH ###
+POSTHOOK: query: select * from default.ice_meta_2.data_files
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@ice_meta_2
+POSTHOOK: Output: hdfs://### HDFS PATH ###
+0	hdfs://### HDFS PATH ###	ORC	0	{"b":"four"}	1	350	{1:6,2:13}	{1:1,2:1}	{1:0,2:0}	{}	{1:	   ,2:four}	{1:	   ,2:four}	NULL	[3]	NULL	0
+0	hdfs://### HDFS PATH ###	ORC	0	{"b":"three"}	3	366	{1:8,2:19}	{1:3,2:3}	{1:0,2:0}	{}	{1:   ,2:three}	{1:   ,2:three}	NULL	[3]	NULL	0
```

Comment:
are these numbers 350,366 and all filesizes? if so we have seen them in past that sometimes they contribute to flakiness, any possibility if we can mask these values?

Reply:
> In general looks good. @zhangbutao can you check what is this problem, one out files ain't visible <img alt="image" width="1770" src="https://user-images.githubusercontent.com/25608848/225461206-9e15d8a2-1cf5-48bf-8772-806df2f62b0c.png">

This problem was introduced by  original PR https://github.com/apache/hive/pull/2589/files#diff-f88561b4ed24634e3b00be85d9b75b58aba93d8455f0d58bb6ec3a201abecb82. I think it is probaly due to the github can not display some complex column type, e.g `map<int,binary>`. 



> are these numbers 350,366 and all filesizes? if so we have seen them in past that sometimes they contribute to flakiness, any possibility if we can mask these values?

 I chosed some fixed column values to avoid flakiness.